src/daft-dsl/src/functions/numeric/mod.rs

```diff
@@ -57,32 +60,32 @@ pub enum NumericExpr {
 impl NumericExpr {
     #[inline]
     pub fn get_evaluator(&self) -> &dyn FunctionEvaluator {
-        use NumericExpr::*;
```

Comment:
What is the reasoning behind removing this and using `NumericExpr::` explicitly?

Reply:
This is more so for peace-of-mind.

Let's say you have:

```rs
enum MyEnum {
  Variant1,
  Variant2,
  Variant3,
}
```

If you do the following:

```rs
use MyEnum::*;

match {
  Variant1 => ...,
  Variant2 => ...,
  Variant3 => ...,
}
```

and someone in the future deletes (or modifies) `Variant3`, then the above match case will continue to compile *without an error* (because rust will then assume that `Variant3` is now a variable binding). The only indication that something is off here is that rust will issue a warning (camel-case warning in this situation). But in my opinion, removing an enum variant should result in a hard error in all match cases that use on that enum.

Reply:
We can prevent against that by either using the full name (or aliasing the enum name if its name is too long).

Reply:
This is a good point! Sounds good then